frontend/website/docs/developers/archive-node.md

```diff
@@ -2,16 +2,30 @@
 
 ## Background
 
-Since Coda nodes are by default succinct, if you need to preserve historical state, you'll want to run an `archive node`. Start the daemon using this command to run an archive process reachable at port `3086`:
-```
-coda daemon -propose-key ~/keys/my_wallet -rest 3085 -archive-port 3086
+Since Coda nodes are by default succinct, if you need to preserve historical state, you'll want to run an `archive node`. The easiest way to run the `archive node` is by the prod archive node docker-compose file.
```

Comment:
```suggestion
Since Coda nodes are by default succinct, if you need to preserve historical state, you'll want to run an `archive node`. The easiest way to run the `archive node` is by using the archive node docker-compose file:
```